scripts/build_collection.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+
+# commands
+DOCKER=${DOCKER:-docker}
+
+# Build collection contract
+WORKSPACE_DIR=$(realpath $(dirname $0)/../)
+
+if [ ! -f "${WORKSPACE_DIR}/artifacts/collection.wasm" ]; then
+  echo "##### Build collection.wasm #####" >&2
+  ${DOCKER} run --rm -v "${WORKSPACE_DIR}":/code \
+    --mount type=volume,source="devcontract_cache_collection",target=/code/examples/contracts/collection/target \
+    --mount type=volume,source=registry_cache,target=/usr/local/cargo/registry \
+    cosmwasm/rust-optimizer:0.15.0 ./examples/contracts/collection
```

Comment:
Is it necessary to add a `arm` option? And I'm worried about whether `0.15.0` version can compile.

Reply:
I confirm that this works with an arm (Apple silicon) chip. But, if you know a better way, please suggest it.
I will add a branch if env `ARCH` is given it will be specified.

Reply:
> I confirm that this works with an arm (Apple silicon) chip.

In workflow, it works without `arm64` option, but if we want to compile locally, I think we need add `arm64`